KDIC/PlayerViewController.swift

```diff
@@ -22,30 +14,15 @@ class PlayerViewController: UIViewController {
         //navigation bar color
         self.navigationController!.navigationBar.barTintColor = UIColor.redColor()
         self.navigationController!.navigationBar.titleTextAttributes = [NSForegroundColorAttributeName : UIColor.whiteColor()]
-        
-        
-        //let kdicScrapperReqs = KdicScrapperReqs(urlStr: "https://kdic-scrapper.herokuapp.com/blogs")
-        //kdicScrapperReqs.makeApiCall()
-        
-        
-    }
-
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+           
     }
-    
 
-    
     @IBAction func didPressPlayer(sender: AnyObject) {
         togglePlayer()
     }
     
-    /**
-     * Player controls
-     **/
     func togglePlayer() {
-        if KdicPlayer.sharedInstance.currentlyPlaying() {
+        if KDICPlayer.sharedInstance.isPlaying {
```

Comment:
Can't you just do `KDICPlayer.sharedInstance.toggle()` here? Then you could also remove the `playRadio()` and `pauseRadio()` functions.